sdk/aqueduct/enums.py

```diff
@@ -81,11 +81,18 @@ class RelationalDBServices(str, Enum, metaclass=MetaEnum):
 
 
 class ExecutionStatus(str, Enum, metaclass=MetaEnum):
+    UNKNOWN = "unknown"
     SUCCEEDED = "succeeded"
     FAILED = "failed"
     PENDING = "pending"
 
 
+class FailureType(Enum, metaclass=MetaEnum):
+    SUCCESS = 0
+    SYSTEM = 1
+    USER = 2
+
```

Comment:
I had some questions about the enum values here, which I wrote about in the prereq PR reviews.